modules/appeals_api/app/swagger/supplemental_claims/v0/swagger_dev.json

```diff
@@ -3219,122 +3219,6 @@
       }
     },
     "schemas": {
-      "address": {
```

Comment:
Looks like the shared schemas are not getting included for the `v0` files anymore.

Reply:
I think I fixed this - the types of the keys used to slice the shared schemas and the contestable issues schemas were wrong - I also figured out that there was a case where the API_NAME could be wrong, so that's fixed too - this seems to really improve the overall diff with master!